wp-redis.php

```diff
@@ -35,21 +35,26 @@
  */
 function wp_redis_get_info() {
 	global $wp_object_cache, $redis_server;
+	// Default Redis port.
+	$port = 6379;
+	// Default Redis database number.
+	$database = 0;
 
 	if ( empty( $redis_server ) ) {
 		// Attempt to automatically load Pantheon's Redis config from the env.
 		if ( isset( $_SERVER['CACHE_HOST'] ) ) {
 			$redis_server = [
-				'host' => sanitize_text_field( $_SERVER['CACHE_HOST'] ),
-				'port' => isset( $_SERVER['CACHE_PORT'] ) ? sanitize_text_field( $_SERVER['CACHE_PORT'] ) : 6379,
-				'auth' => isset( $_SERVER['CACHE_PASSWORD'] ) ? sanitize_text_field( $_SERVER['CACHE_PASSWORD'] ) : null,
-				'database' => isset( $_SERVER['CACHE_DB'] ) ? sanitize_text_field( $_SERVER['CACHE_DB'] ) : 0,
+				'host' => strip_tags( $_SERVER['CACHE_HOST'] ),
```

Comment:
Same comments as above about `wp_strip_all_tags` vs `strip_tags` and explicitly ignoring linting rules.